packages/ui/src/memberships/components/MemberProfile/MemberRoleToggle.tsx

```diff
@@ -79,10 +91,13 @@ export const MemberRoleToggle = ({ role }: MemberRoleToggleProps) => {
                     </SidePaneColumn>
                   </SidePaneRow>
                   <ButtonsGroup align="left">
-                    <ButtonGhost size="small">
+                    <ButtonGhost size="small" onClick={showApplicationModal}>
                       Application preview <Arrow direction="right" />
                     </ButtonGhost>
-                    <ButtonGhost size="small">
+                    <ButtonGhost
```

Comment:
Could you use `LinkButton` instead of `<ButtonGhost>` + `history.push()`.

Reply:
Done, it works. But style for LinkButtonGhost differs from ButtonGhost